.github/stale.yml

```diff
@@ -0,0 +1,20 @@
+# Stale issue bot: https://github.com/marketplace/stale
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 30
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - RFC
```

Comment:
I think there should be more. Actually, can we use a black list strategy? e.g., if an issue does not have any label or it is under triage but is inactive for 7 days, we move it to stale.